cluster/plan.go

```diff
@@ -575,6 +597,7 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 			"/dev:/host/dev:rprivate",
 			"/var/log/containers:/var/log/containers:z",
 			"/var/log/pods:/var/log/pods:z",
+			"/var/log/calico/cni:/var/log/calico/cni:z",
```

Comment:
We're conditionally adding what appears to be the same bind. Should it be removed from here then?

Reply:
Yes that is correct. Seems like it was added by mistake. Removed the bind.